jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties

```diff
@@ -34,6 +34,9 @@ Process_Name=Process Name
 Process_Instance_Id=Process Instance Id
 Process_Definition_Id=Process Definition Id
 Process_Instance_Details=Process Instance Details
+Process_Instance_Correlation_Key=Process instance correlation key
```

Comment:
Since the correlation key is really associated with a process instance, I find _Process Correlation Key_ a bit confusing. If you consider shortening for the sake of shortness for UI display, I'd prefer _Correlation Key_ @cristianonicolai WDYT?